### PR TITLE
Adds current config in Recommendations

### PR DESCRIPTION
**Pull request by bharathappali**
As per the requirement from a user experience standpoint, This PR adds the ability to send current CPU and Memory Requests/Limits as part of the recommendations object. 

Doc changes need to be added.

### CODE REVIEW

**Comment by bharathappali** [comment]
Sample Output of `listRecommendations`

```
[
  {
    "cluster_name": "cluster-one-division-bell",
    "kubernetes_objects": [
      {
        "type": "deployment",
        "name": "tfb-qrh-deployment_0",
        "namespace": "default_0",
        "containers": [
          {
            "container_image_name": "kruize/tfb-qrh:1.13.2.F_et17",
            "container_name": "tfb-server-1",
            "recommendations": {
              "notifications": {
                "112101": {
                  "type": "info",
                  "message": "Duration Based Recommendations Available",
                  "code": 112101
                }
              },
              "data": {
                "2023-04-02T13:30:00.680Z": {
                  "duration_based": {
                    "short_term": {
                      "monitoring_start_time": "2023-04-01T12:00:00.000Z",
                      "monitoring_end_time": "2023-04-02T13:30:00.680Z",
                      "duration_in_hours": 24.0,
                      "pods_count": 27,
                      "confidence_level": 0.0,
                      "current": {
                        "requests": {
                          "memory": {
                            "amount": 50.21,
                            "format": "MiB"
                          },
                          "cpu": {
                            "amount": 5.37,
                            "format": "cores"
                          }
                        },
                        "limits": {
                          "memory": {
                            "amount": 100.0,
                            "format": "MiB"
                          },
                          "cpu": {
                            "amount": 0.5,
                            "format": "cores"
                          }
                        }
                      },
                      "config": {
                        "requests": {
                          "memory": {
                            "amount": 238.2,
                            "format": "MiB"
                          },
                          "cpu": {
                            "amount": 0.9299999999999999,
                            "format": "cores"
                          }
                        },
                        "limits": {
                          "memory": {
                            "amount": 238.2,
                            "format": "MiB"
                          },
                          "cpu": {
                            "amount": 0.9299999999999999,
                            "format": "cores"
                          }
                        }
                      },
                      "variation": {
                        "requests": {
                          "memory": {
                            "amount": 187.98999999999998,
                            "format": "MiB"
                          },
                          "cpu": {
                            "amount": -4.44,
                            "format": "cores"
                          }
                        },
                        "limits": {
                          "memory": {
                            "amount": 138.2,
                            "format": "MiB"
                          },
                          "cpu": {
                            "amount": 0.42999999999999994,
                            "format": "cores"
                          }
                        }
                      },
                      "notifications": {}
                    },
                    "medium_term": {
                      "pods_count": 0,
                      "confidence_level": 0.0,
                      "notifications": {
                        "120001": {
                          "type": "info",
                          "message": "There is not enough data available to generate a recommendation.",
                          "code": 120001
                        }
                      }
                    },
                    "long_term": {
                      "pods_count": 0,
                      "confidence_level": 0.0,
                      "notifications": {
                        "120001": {
                          "type": "info",
                          "message": "There is not enough data available to generate a recommendation.",
                          "code": 120001
                        }
                      }
                    }
                  }
                }
              }
            }
          },
          {
            "container_image_name": "kruize/tfb-db:1.15",
            "container_name": "tfb-server-0",
            "recommendations": {
              "notifications": {
                "120001": {
                  "type": "info",
                  "message": "There is not enough data available to generate a recommendation.",
                  "code": 120001
                }
              },
              "data": {}
            }
          }
        ]
      }
    ],
    "version": "1.0",
    "experiment_name": "quarkus-resteasy-kruize-min-http-response-time-db_0"
  }
]

```

File: src/main/java/com/autotune/analyzer/utils/AnalyzerConstants.java [hunk 1]
```diff
@@ -138,6 +138,22 @@ public enum PersistenceType {
...
+
+        private RecommendationSection(String name) {
+            this.name = name;
+        }
+
+        public String getName() {
+            return this.name;
+        }
+    }
+
```
Comment (dinogun) [inline src/main/java/com/autotune/analyzer/utils/AnalyzerConstants.java:156]: I'd suggest move all of the recommendations constants to the `recommendations` dir. This is bloating the `AnalyzerConstants` even though none of this is used beyond the `recommendations` package

File: src/main/java/com/autotune/analyzer/recommendations/engine/DurationBasedRecommendationEngine.java [hunk 2]
```diff
@@ -83,181 +446,106 @@ public HashMap<String, Recommendation> generateRecommendation(ContainerData cont
...
+                        recommendation,
+                        notifications,
+                        internalMapToPopulate,
+                        numPods,
+                        hours
+                );
+
+                // Use success in future if you want to avoid sending the recommendations
+                if (!isSuccess) {
+                    // Handle the issue accordingly
```
Comment (dinogun) [inline src/main/java/com/autotune/analyzer/recommendations/engine/DurationBasedRecommendationEngine.java:542]: I think we need to add notifications for the rest as well and not have to resort to fail the API

File: src/main/java/com/autotune/analyzer/utils/AnalyzerConstants.java (hunk 1 above)
Comment (bharathappali) [reply src/main/java/com/autotune/analyzer/utils/AnalyzerConstants.java:156]: Thanks @dinogun for the review, Will make the changes and push the commit to PR.

File: tests/scripts/remote_monitoring_tests/helpers/list_reco_json_schema.py [hunk 3]
```diff
@@ -395,6 +479,174 @@
...
+                                            "required": [
+
+                                            ]
+                                          }
+                                        },
+                                        "required": [
+
+                                        ]
+                                      },
+                                      "current": {
```
Comment (chandrams) [inline tests/scripts/remote_monitoring_tests/helpers/list_reco_json_schema.py:566]: @bharathappali - Current section is repeated again?

**Review by dinogun — APPROVED** [review]
LGTM. Some of the changes suggested will be handled in #864 as this will enable testing

File: tests/scripts/remote_monitoring_tests/helpers/list_reco_json_schema.py (hunk 3 above)
Comment (bharathappali) [reply tests/scripts/remote_monitoring_tests/helpers/list_reco_json_schema.py]: Removed the duplicate entries. Thanks for review @chandrams

**Pull request merged**